store/postgres/src/filter.rs

```diff
@@ -271,9 +271,72 @@ fn store_filter_by_mode<'a>(
         }
         // Is `attribute` equal to some `v` in `query_values`?
         StoreFilter::In(attribute, query_values) => {
-            query_values.into_iter().try_fold(query, |q, v| {
-                store_filter_by_mode(q, StoreFilter::Equal(attribute.clone(), v), FilterMode::Or)
-            })?
+            let op = " = ANY (";
+            match query_values[0].clone() {
+                Value::Bool(_) => add_filter(
```

Comment:
Do we have a task to track query validation? Unless we validate the values in the query, we'll have to assume that `query_values` could be a list with values of different types and then the `bind::<Array<Bool>, _>(query_values)` will trigger `unimplemented!()` and panic.

Reply:
An easy check we could perform upfront is whether all values have the same type (can be checked using a simple reduction over all values).

Can we create a task to do this and allow filter generation to fail and return an error back to the client in case it does?

Reply:
I created issue #272 to track validation of `query_values`.